### ISSUE TITLE
Display Settings Can't Be Changed From A Remote Sesson

### ISSUE THREAD

**Issue by adamyala** [DanielThomas/ec2gaming#10]
I'm going through the first time configuration. I am at:
```
Right-click the desktop, select Screen resolution and select the highest available resolution
```
When I the window opens it says:
```
The display settings can't be changed from a remote session.
```
Did I mess something up when editing registries?
<img width="838" alt="screen shot 2016-11-05 at 2 02 24 pm" src="https://cloud.githubusercontent.com/assets/3434677/20032810/83cd35d8-a360-11e6-9901-52fb4f3c2864.png">

**Comment by DanielThomas** [comment]
The step immediately before that says to connect with VNC :)

**Comment by adamyala** [comment]
You're correct, my bad.

When I run `ec2gaming vnc`, the Screen Sharing app starts running, says:

```
Connecting to "10.8.0.1"
```

Then I get the error:

```
Connection failed to "10.8.0.1"

Unable to communicate with "10.8.0.1". Make sure the remote computer is available and the firewall is not blocking screensharing.
```

I went into the ec2gaming security group and allowed all TCP traffic and still got this error. Below is a screenshot of what the TightVNC service is running. I also didn't know if the remote would need access to mine machine so I went into System Preferences -> Sharing, and enabled Screen Sharing. Still no dice. I also went into TightVNC to unset/set my password. Any advice?

<img width="176" alt="screen shot 2016-11-05 at 8 53 31 pm" src="https://cloud.githubusercontent.com/assets/3434677/20034877/02d48d54-a39d-11e6-8afe-9c2f88ca84d9.png">
